.licenserc.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright VMware, Inc.
+# SPDX-License-Identifier: APACHE-2.0
+
+---
+header:
+  # License format for Helm templates
+  - comment: on-failure
+    license:
+      spdx-id: Apache-2.0
+      copyright-owner: VMware, Inc.
+
+      content: |
+        {{- /*
+        Copyright VMware, Inc.
+        SPDX-License-Identifier: APACHE-2.0
+        */}}
+
+    paths:
```

Comment:
There are also a `_helpers.tpl` file present in most Helm charts that would be nice to check as well, see https://github.com/bitnami/charts/blob/main/bitnami/postgresql/templates/_helpers.tpl

Reply:
Perfect, totally fine